delfin/common/config.py

```diff
@@ -100,6 +100,16 @@
 CONF.register_opts(global_opts)
 
 
+security_opts = [
+    cfg.StrOpt('verify',
```

Comment:
As of now this config is global to all backends. We need to find a way to specify this for driver/backend .. Have created an issue to track https://github.com/sodafoundation/delfin/issues/306